cinema/management/commands/wait_for_db.py

```diff
@@ -0,0 +1,24 @@
+import time
+from django.db import connections
+from django.db.utils import OperationalError
+from django.core.management.base import BaseCommand
+
+
+class Command(BaseCommand):
+    def handle(self, *args, **options):
+        self.stdout.write("Wait for the database to be available...")
+        db_connect = None
+        max_attempts = 10
+        attempts = 0
+        while not db_connect:
+            try:
+                db_connect = connections["default"].ensure_connection()
+            except OperationalError:
+                attempts += 1
+                self.stdout.write("Database unavailable, waiting 1 second...")
+                time.sleep(1)
+
```

Comment:
In your while loop, you are not checking if the maximum number of attempts has been reached. You need to add a condition to break the loop if `attempts` is equal or greater than `max_attempts`. Otherwise, if the database is not available, your application will be stuck in an infinite loop.